packages/mediapipe-core/lib/src/io/test_utils.dart

```diff
@@ -56,3 +58,47 @@ void populateClassifications(
   classifications.head_name = headName.copyToNative();
   classifications.head_index = headIndex;
 }
+
+/// Hydrates a faked [core_bindings.Embedding] object.
+void populateEmbedding(
+  core_bindings.Embedding embedding, {
+  bool quantize = false,
+  bool l2Normalize = false,
```

Comment:
Same here re: asserting one or the other is false, or using EmbeddingType enum.

Reply:
Responded in [this comment](https://github.com/google/flutter-mediapipe/pull/21#discussion_r1546734665) above.